src/TraceEvent/Computers/TraceManagedProcess.cs

```diff
@@ -121,6 +121,11 @@ public class TraceLoadedDotNetRuntime
         /// Returns the .NET startup flags
         /// </summary>
         public StartupFlags StartupFlags { get; internal set; }
+        /// <summary>
+        /// Date and time of when the runtime was built
+        /// This is useful when a more detailed version is not present
+        /// </summary>
+        public DateTime RuntimeBuiltTime { get; internal set; }
```

Comment:
Non-nullable to align with other DateTime types within TraceEvent.  It is a DateTime merely since that is the underlying type present in the ETL file.